examples/plot_sleep_staging_usleep.py

```diff
@@ -209,6 +209,7 @@
     with_skip_connection=True,
     n_classes=n_classes,
     input_size_s=input_size_samples / sfreq,
+    apply_softmax=True
```

Comment:
you need to use nn.CrossEntropyLoss here as you use class-weights? if so can you add a comment
so it's clear why you need to change a default parameter in a first example?

Reply:
what about these questions @Tgnassou ? please do add some comment about the `depth=12` line before the model creation, so it shows up clearly in the example doc why we reduce here. I am also confused about `apply_softmax=True` and using `nn.CrossEntropyLoss` as doc of cross entropy loss states:
https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

```
This criterion combines LogSoftmax and NLLLoss in one single class.

[...]

The input is expected to contain raw, unnormalized scores for each class.```

Reply:
Did you see this comment here @hubertjb ?